project_phase/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2018 Oihane Crucelaegui - AvanzOSC
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    "name": "Project Phase",
+    "version": "11.0.1.1.0",
```

Comment:
@dalonsod If a new Module then` "version": "11.0.0.0.0",`

And Add `Summary` key

Reply:
@nikul-serpentcs He has to change it to "11.0.1.0.0" and summary is not mandatory. The thing is that this module has been migrated from other repo.

Reply:
As said [here](https://github.com/OCA/account-invoicing/pull/533#discussion_r288586399) I've only changed version number to 11.0.1.0.0